authgateway/src/main/resources/application.yml

```diff
@@ -1,20 +1,4 @@
-eureka:
-  client:
-    serviceUrl:
-      defaultZone: http://localhost:8761/eureka/
-  instance:
-    leaseRenewalIntervalInSeconds: 10
-    metadataMap:
-      instanceId: ${spring.application.name}:${spring.application.instance_id:${server.port}}
-spring:
-  application.name: blueorgreenauthgateway
-  config:
-    import: "optional:configserver:http://localhost:8888"
-management:
-  endpoints:
-    web:
-      exposure:
-        include: "*"
-  endpoint:
-    shutdown:
-      enabled: true
\ No newline at end of file
+spring.application.name: blueorgreenauthgateway
+spring.config.import: "optional:configserver:http://localhost:8888"
+
+wavefront.application.name: color-app
```

Comment:
This is not the name of the service, this is the name of the whole group of services in wavefront.